### PR TITLE
chore(ci): add steps to generate the lib types for every release

### PR DESCRIPTION
**Pull request by imrishabh18**
Issue - #172

### CODE REVIEW

**Review by muhammadsammy — COMMENTED** [review]
That should do it but, it won't commit the changes made by `yarn generate` to git.
I would suggest you add another step to commit these changes using   https://github.com/marketplace/actions/add-commit or https://github.com/marketplace/actions/git-auto-commit or any other similar actions

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -23,6 +23,13 @@ jobs:
           registry-url: 'https://registry.npmjs.org'
         if: ${{ steps.release.outputs.release_created }}
       - run: yarn install --frozen-lockfile
+      - name: Generating Lib types
+        run: yarn generate
+      - name: Commit yarn generated files
+        uses: stefanzweifel/git-auto-commit-action@v2.3.0
+        with:
+          commit_message: Commit yarn generated files
```
Comment (muhammadsammy) [inline .github/workflows/release.yml:31]: Please change this commit message to something like this "chore: generate library types" to be compatible with the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) spec

**Review by muhammadsammy — CHANGES_REQUESTED** [review]
Everything looks good but please change the commit message in auto-commit action to something like: 
`chore: generate library types` to be compatible with the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) spec. Thank you!

**Review by muhammadsammy — COMMENTED** [review]
Also, please change your commit messages to start with `chore:` or `chore(ci):` prefix 
to be compatible with Conventional Commits so I can rebase merge your PR instead of squashing your commits

**Review by muhammadsammy — APPROVED** [review]
That's great. Thank you, @imrishabh18 !
Could you please rewrite/rename your commit messages to adhere with the Conventional Commits spec or would you rather me to squash them into one commit?

**Comment by imrishabh18** [comment]
Sorry! Isn't this the conventional commit `chore: rename commit message ` style? Maybe I missed something you can squash the commits

**Comment by muhammadsammy** [comment]
I was referring to other commits in your commit history in this PR, 7ad38c3aa0afd022b392923844619fa911bb3057 and bf283e6a4a9d706b8ea05e44399c661297495b37 are not adhering to the conventional commit spec so I can't rebase them into master.

Thanks again for contributing :smile:

**Pull request merged**